src/applications/mhv-medical-records/containers/AllergyDetails.jsx

```diff
@@ -136,10 +190,10 @@ Provider notes: ${allergy.notes} \n`;
             aria-describedby="allergy-date"
```

Comment:
Should the surrounding condition be `if (allergyData)` instead of `if (allergy)`? Not sure if it matters

Reply:
it did matter. it was blur, but if I did `allergyData` then it broke the whole page. It shouldn't matter in practice but its definately code smell.

Reply:
Fixed this smell. 

It was that allergyData originally was always truthy, but the page logic here expected the possibility that it could be falsey. I added some logic to allow allergyData to be falsey